Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:18 AS build
+
+WORKDIR /opt/node_app
+
+COPY package.json yarn.lock ./
+RUN yarn --ignore-optional --network-timeout 600000
+
+ARG NODE_ENV=production
+
+COPY . .
+RUN yarn build
+
+FROM nginx:1.21-alpine
+
+COPY --from=build /opt/node_app/dist /opt/node_app/index.html /usr/share/nginx/html/
```

As this will be a playground image, we need to change the command to yarn build:playground, and copying of the files needs to be done from the /public folder (bundle will correctly make chunks for current library). This should solve the image problem.

```suggestion
RUN yarn build:playground

FROM nginx:1.21-alpine

COPY --from=build /opt/node_app/public /usr/share/nginx/html/
```

---

@rgarrigue can you try applying the suggested changes by @igorwessel above and it should work